specs/carves.table

```diff
@@ -1,13 +1,13 @@
 table_name("carves")
 description("Forensic Carves.")
 schema([
-    Column("time", BIGINT, "Time at which the carve was kicked off"),
```

Comment:
Why are these all set to `additional`?

I only see `path` and `carve` being used in the table implementation.